Source/WebKit/WebProcess/WebPage/WKAccessibilityWebPageObjectIOS.mm

```diff
@@ -74,8 +74,15 @@ - (id)accessibilityHitTest:(NSPoint)point
 {
     if (!m_page)
         return nil;
-    
+
     WebCore::IntPoint convertedPoint = m_page->accessibilityScreenToRootView(WebCore::IntPoint(point));
+
+    // If we are hit-testing a remote element, offset the hit test by the scroll of the web page.
+    if (NakedPtr remoteLocalFrame = [self remoteLocalFrame]) {
+        if (auto* frameView = remoteLocalFrame->view())
```

Comment:
We call `scrollPosition()` on this `frameView`, which is not a [trivial](https://github.com/WebKit/WebKit/wiki/Safer-CPP-Guidelines#when-calling-a-non-trivial-member-function-on-an-object-hold-a-smart-pointer-to-the-object-on-the-stack) function, so I think we have to use a smart pointer here. Looks like `CheckedPtr` is our only option for `FrameView`s.